docs/source/Evaluation_Tutorial.md

```diff
@@ -251,7 +185,37 @@ docker run \
   -t unitytechnologies/datasetinsights:0.2.0
 ```
 
-### Using our pre-trained models
+## Part 5 (Optional): Fine-tune synthetic model on a real-world dataset
```

Comment:
I like the flow you outlined here which pushed optional training steps here. However, I think both `Train on UnityGroceries-RealWorld dataset` and  `Train on synthetic dataset generated on Unity Simulation` should be moved to this section, as they are not the default path that we want users to follow. Otherwise, the model training part became too tedious to follow.

Maybe we can have this section `## Part 5 (Optional): Optional Training Pipelines` and make four sections (`Train on UnityGroceries-RealWorld dataset`,  `Train on synthetic dataset generated on Unity Simulation` and `Fine-tune synthetic model on a real-world dataset`)under part 5 here?

Reply:
Done!